Makefile

```
@@ -775,9 +776,8 @@ docopen: $(apidocs_html)
 	@$(PYTHON) -mwebbrowser file://$(PWD)/out/doc/api/all.html
 
 .PHONY: docserve
-docserve: $(apidocs_html)
-	@$(PYTHON) -mwebbrowser http://localhost:8000/all.html
-	@$(PYTHON) -m http.server -d $(PWD)/out/doc/api
+docserve:
```

Isn't it better to include apidocs_html as a prerequisite? So if you run `make docserve` and you haven't built the docs, it will build them?

```suggestion
docserve: $(apidocs_html)
```

---

Ah, you're right. Confirmed this by looking at how the `jekyll serve` command operates for comparison.

---

Hmm, I just checked the Makefile and I think we were mistaken about what this line is actually doing. It definitely *doesn't* build the docs.